dspace-api/src/main/java/org/dspace/importer/external/scopus/service/LiveImportClient.java

```diff
@@ -0,0 +1,21 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.importer.external.scopus.service;
+
+import java.util.Map;
+
+/**
+ * Interface for classes that allow to contact LiveImport clients.
+ *
+ * @author Mykhaylo Boychuk (mykhaylo.boychuk at 4science.com)
+ */
+public interface LiveImportClient {
```

Comment:
This Interface & the corresponding implementation do not appear to be specific to Scopus.  Maybe they belong in a different package?  Currently it seems odd they are under the `...scopus.service` package.  Maybe they should move to `org.dspace.importer.external.service`?